src/field_theory/ax_grothendieck.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2023 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import data.mv_polynomial.basic
+import ring_theory.algebraic
+import data.fintype.card
+
+/-!
+# Ax-Grothendieck for algebraic extensions of `zmod p`
+
+This file proves that if `R` is an algebraic extension of a finite field,
+then any injective polynomial map `R^n -> R^n` is also surjective.
+
+This proof is required for the true Ax-Grothendieck theorem, which proves the same result
+for any algebraically closed field of characteristic zero. The proof of the theorem for
+characteristic zero is not in mathlib, but it is at https://github.com/Jlh18/ModelTheoryInLean8
```

Comment:
Can you make that into a TODO to be sure we update it when the next PR comes?
```suggestion
for any algebraically closed field of characteristic zero.

## TODO
The proof of the theorem for characteristic zero is not in mathlib, but it is at
https://github.com/Jlh18/ModelTheoryInLean8
```
Also, wow! Us mortals are still on Lean 4.